src/com/vivian/java/leetcode/Q383_RansomNote.java

```diff
@@ -13,5 +13,29 @@
  * canConstruct("aa", "aab") -> true
  */
 public class Q383_RansomNote {
+    public boolean ransonNote(String mag, String str) {
```

Comment:
The naming of your parameters is actually reversed. The magazine is supposed to be the bigger string.